clouddriver/clouddriver-web/src/main/java/com/netflix/spinnaker/clouddriver/controllers/ArtifactController.java

```diff
@@ -73,7 +73,9 @@ List<ArtifactCredentials> list() {
   }
 
   // PUT because we need to send a body, which GET does not allow for spring/retrofit
-  @RequestMapping(method = RequestMethod.PUT, value = "/fetch")
+  @RequestMapping(
```

Comment:
Since this is an internal (i.e. not gate) endpoint, it seems like the better fix would be to change the caller.

Reply:
@dbyron-sf please take a look at this commit for the change https://github.com/rahul-chekuri/spinnaker/commit/d528d879